src/website/views/components/Polaris/PolarisContainer.tsx

```diff
@@ -22,88 +22,92 @@ function stringToParagraphs(text: string) {
-      {
-        title: 'GPS',
-        data: ['GPS', '12 hours', 'JSON'],
-        action: downloadGPSData,
-      },
-      {
-        title: 'AIS Ships',
-        data: ['AIS Ships', '12 hours', 'JSON'],
-        action: downloadAISShipsData,
-      },
-      {
-        title: 'Global Path',
-        data: ['Global Path', '12 hours', 'JSON'],
-        action: downloadGlobalPathData,
-      },
-      {
-        title: 'Local Path',
-        data: ['Local Path', '12 hours', 'JSON'],
-        action: downloadLocalPathData,
-      },
-      {
-        title: 'Batteries',
-        data: ['Batteries', '12 hours', 'JSON'],
-        action: downloadBatteriesData,
-      },
-      {
-        title: 'Wind Sensors',
-        data: ['Wind Sensors', '12 hours', 'JSON', 'Download'],
-        action: downloadWindSensorsData,
-      },
-      {
-        title: 'Generic Sensors',
-        data: ['Generic Sensors', '12 hours', 'JSON', 'Download'],
-        action: downloadGenericSensorsData,
-      },
-    ];
+  useEffect(() => {
+    fetch('/info/PolarisVoyageDescription.txt')
+      .then((res) => res.text())
+      .then((data) => setDescription(data));
+  }, []);
 
-    return (
-      <div className={styles.container}>
-        <div className={styles.spatialCoverageContainer}>
-          <div className={styles.spatialCoverageTitle}>Polaris</div>
-          <div className={styles.coverageItem}>
-            <div>Latitudes</div>
-            <div className={styles.coverageContent}>Hello1</div>
-          </div>
-          <div className={styles.coverageItem}>
-            <div>Longitudes</div>
-            <div className={styles.coverageContent}>Hello2</div>
-          </div>
-          <div className={styles.coverageItem}>
-            <div>Date Range</div>
-            <div className={styles.coverageContent}>Hello3</div>
-          </div>
-          <div className={styles.coverageItem}>
-            <div>Geography</div>
-            <div className={styles.coverageContent}>Hello4</div>
-          </div>
+  const paragraphs = description
+
+  const customContents = [
+    {
+      title: 'GPS',
+      data: ['GPS', '12 hours', 'JSON'],
+      action: downloadGPSData,
+    },
+    {
+      title: 'AIS Ships',
+      data: ['AIS Ships', '12 hours', 'JSON'],
+      action: downloadAISShipsData,
+    },
+    {
+      title: 'Global Path',
+      data: ['Global Path', '12 hours', 'JSON'],
+      action: downloadGlobalPathData,
+    },
+    {
+      title: 'Local Path',
+      data: ['Local Path', '12 hours', 'JSON'],
+      action: downloadLocalPathData,
+    },
+    {
+      title: 'Batteries',
+      data: ['Batteries', '12 hours', 'JSON'],
+      action: downloadBatteriesData,
+    },
+    {
+      title: 'Wind Sensors',
+      data: ['Wind Sensors', '12 hours', 'JSON', 'Download'],
+      action: downloadWindSensorsData,
+    },
+    {
+      title: 'Generic Sensors',
+      data: ['Generic Sensors', '12 hours', 'JSON', 'Download'],
```

Comment:
Please remove 'Download' from the data array for Wind Sensors and Generic Sensors.